controllers/projects_controller.go

```diff
@@ -158,7 +175,53 @@ func (u *Projects) Update(c web.C, w http.ResponseWriter, r *http.Request) {
 	if repo != nil {
 		repositoryId = repo.Id
 	}
-	jsonProject := ProjectJsonFormat{Id: project.Id, UserId: project.UserId, Title: project.Title, Description: project.Description, RepositoryID: repositoryId}
+	jsonProject := ProjectJsonFormat{Id: project.Id, UserId: project.UserId, Title: project.Title, Description: project.Description, ShowIssues: project.ShowIssues, ShowPullRequests: project.ShowPullRequests, RepositoryID: repositoryId}
+	encoder.Encode(jsonProject)
+}
+
+func (u *Projects) Settings(c web.C, w http.ResponseWriter, r *http.Request) {
```

Comment:
__[golint]__
* [golint] exported method Projects.Settings should have comment or be unexported